packages/client-twitter/src/interactions.ts

```diff
@@ -448,7 +448,8 @@ export class TwitterInteractionClient {
                 await this.runtime.processActions(
                     message,
                     responseMessages,
-                    state
+                    state,
+                    callback
```

Comment:
Is this a bug fix?

Reply:
yes it is

Reply:
yeah the callback argument was missing here